python/model_hosting_container_standards/supervisor/generator.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
Let's make this logic more extensible. Let's allow customers to set the supervisor configuration at runtime as they choose. 

ENV SUPERVISOR_SECTION_KEY=VALUE

should translate to 

```
[SECTION]
KEY=VALUE
```

Reply:
We should definitely continue to provide a minimal configuration that is guaranteed to work, but no need to restrict user customization.